prov/verbs/src/ep_rdm/verbs_utils.h

```diff
@@ -217,11 +218,11 @@ do {                                                                        \
         case FI_LOG_WARN:                                                   \
         case FI_LOG_TRACE:                                                  \
         case FI_LOG_INFO:                                                   \
-            fprintf(stderr, str);                                           \
+            fprintf(stderr, "%s", str);                                     \
             break;                                                          \
         case FI_LOG_DEBUG:                                                  \
         default:                                                            \
-            VERBS_DBG(str);                                                 \
+            VERBS_DBG(FI_LOG_EP_DATA, "%s", str);                           \
             break;                                                          \
     }                                                                       \
 } while (0);
```

Comment:
This isn't related to this change, but this macro isn't correct.  It needs to use the logging functions, not call fprintf.